Adjusted_Districts_Pop_09_28_22/README.md

```diff
@@ -0,0 +1,46 @@
+# 2022 Districts using Adjusted Data - Populations 09_28_22
+
+## Background:
+- We received a data request asking for total populations of the 2022 districts.
+- Although most states draw their redistricting plans using the census' population, a handful of states use adjusted data.
+- The usage of adjusted data is made more complicated by the fact that not every state that produces adjusted data uses it for all levels of redistricting.
+- Below is a list of the states that produce adjusted data and the level(s) of redistricting they use it for:
+    - CA (Congressional and State Legislative)
+    - CO (State Legislative)
+    - CT (State Legislative)
+    - DE (State Legislative)
+    - HI (State Legislative)
+    - MD (Congressional and State Legislative)
+    - MT (State Legislative)
+    - NJ (Congressional and State Legislative)
+    - NV (Congressional and State Legislative)
+    - NY (State Legislative)
+    - PA (State Legislative)
+    - RI (Congressional and State Legislative)
+    - VA (Congressional and State Legislative)
+    - WA (Congressional and State Legislative)
+- Due to this nuance, we thought it would be easier to produce a dataset with the districts that used adjusted data and their adjusted population, rather than adding in an "adjusted population" column to the national block-assignment file.
```

Comment:
"Due to this nuance, we thought it would"  make sense "to produce a dataset with the districts that used adjusted data and their adjusted population, rather than adding in an "adjusted population" column to the national block-assignment file" where the adjusted population would not be relevant in all cases.

Reply:
I just made those changes, can you take a look now?